### DISCUSSION TITLE
shorts ideas ?

### DISCUSSION THREAD

**Discussion by maxandersen** [quarkusio/quarkus#34538] [Community]
Was chatting with @saumya1singh about ideas for short videos (i.e. youtube.com/quarkusio/short) we could make around Quarkus and cross-post on various social medias. 30s to 2min long videos highlighting some fun fact or feature in and around Quarkus.

We agreed it could be interesting to gather ideas in and from the community present here - so that's what this thread is about - post an idea to short or maybe a question you think could be interesting to get an answer to and then maybe @saumya1singh or others who want to try it out make it happen.

I'll start off with my current pool of ideas that I call "the little things in devmode":

* 'w' (web), 'd' (debug) and recently 'x' (open project or last exception in IDE)
* Quarkus does not need a main method but you can add a main method to any example we have and run it from your IDE with devmode (working on making that default #34108)
* terminal mode in devmode
* cli's ability to execute java as scripts within your project

some more conceptual ones are:

* outline the quarkus way of moving things to build to gain speed
* explain how quarkus approach to reactive is about reusing the type safe models/api so you can share or migrate code between reactive and imperative easily 
* highlight the expansion of the Quarkiverse Hub and that everyone can participate

in area of persistence:

* explain and show statelessSession for performance / easy and direct SQL mapping without a stateful entitymanager

other ideas?

**Comment by bulldog98** [comment]
a short introduction into the idea of reactive programming
